app/Http/Controllers/DeviceController.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Facades\DeviceCache;
+use App\Models\Device;
+use App\Models\Vminfo;
+use Auth;
+use Carbon\Carbon;
+use Gate;
+use Illuminate\Http\Request;
+use Illuminate\Support\Arr;
+use LibreNMS\Config;
+use LibreNMS\Util\Url;
+
+class DeviceController extends Controller
+{
+    private $tabs = [
+        'overview' => \App\Http\Controllers\Device\Tabs\OverviewController::class,
+        'graphs' => \App\Http\Controllers\Device\Tabs\GraphsController::class,
+        'health' => \App\Http\Controllers\Device\Tabs\HealthController::class,
+        'apps' => \App\Http\Controllers\Device\Tabs\AppsController::class,
+        'processes' => \App\Http\Controllers\Device\Tabs\ProcessesController::class,
+        'collectd' => \App\Http\Controllers\Device\Tabs\CollectdController::class,
+        'ports' => \App\Http\Controllers\Device\Tabs\PortsController::class,
+        'slas' => \App\Http\Controllers\Device\Tabs\SlasController::class,
+        'wireless' => \App\Http\Controllers\Device\Tabs\WirelessController::class,
+        'accesspoints' => \App\Http\Controllers\Device\Tabs\AccessPointsController::class,
+        'latency' => \App\Http\Controllers\Device\Tabs\LatencyController::class,
```

Comment:
this should probably be moved down, menu order doesnt make sense

Reply:
@Jellyfrog the menu order is based on the existing order.  I think re-arranging the tabs should be a different PR.

Reply:
Latency is not there in master, ie performance is the last option, iirc.

Reply:
Performance is renamed to Latency, so it is in the place of the old performance option.  The old Latency tab is removed.

Reply:
Yes I know, and now "Latency" is in the middle of the menu, instead of being almost at the end.

Old:

![image](https://user-images.githubusercontent.com/759887/80191275-bfe34800-8615-11ea-9368-69f982ed69f5.png)

New:
![image](https://user-images.githubusercontent.com/759887/80191192-a6da9700-8615-11ea-9ff5-8618661a966b.png)

Reply:
Does Ping make sense there?  Apparently I put it in the Ping position (since it replaces that too.
I'm fine with moving it.